src/main/java/stringcalulator/model/Operator.java

```diff
@@ -0,0 +1,47 @@
+package stringcalulator.model;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+import static stringcalulator.model.Constant.Exception.*;
+
+public enum Operator {
+    PLUS("+") {
+        public double operate(double x, double y) {
+            return x + y;
+        }
+    },
+    MINUS("-") {
+        public double operate(double x, double y) {
+            return x - y;
+        }
+    },
```

Comment:
```suggestion
    PLUS("+", Integer::sum),
    MINUS("-", (x, y) -> x - y),
```
로직이 간단하네요! BiFunction을 사용해서 람다 표현식으로도 사용할 수 있습니다!